doc/rule_reference.md

```diff
@@ -4,7 +4,11 @@ The Jary language shares a similar syntax with another language called Yara. But
 The grammar is very simple, you only need to know these concepts: 
 - `import` 
 - `ingress` 
-- `rule` 
+- `rule`
```

Comment:
This part only list high level concepts. The `condition`, `match` and etc is implied by the `section` concept, and should not be mentioned here.